services/device-connection/src/main/java/org/eclipse/hono/deviceconnection/infinispan/RemoteCacheBasedDeviceConnectionService.java

```diff
@@ -89,19 +87,22 @@ protected void doStart(final Future<Void> startFuture) {
         final Promise<Void> result = Promise.promise();
         result.future().setHandler(startFuture);
 
-        if (cacheManager == null) {
+        if (cache != null) {
+            result.complete();
+        } else if (cacheManager == null) {
             result.fail(new IllegalStateException("cache manager is not set"));
         } else {
             context.executeBlocking(r -> {
                 cacheManager.start();
-                cache = cacheManager.getCache("device-connection");
-                cache.start();
+                final BasicCache<String, String> remoteCache = cacheManager.getCache("device-connection");
+                remoteCache.start();
+                cache = new HotrodBasedDeviceConnectionInfoCache(remoteCache);
                 r.complete(cacheManager);
             }, attempt -> {
                 if (attempt.succeeded()) {
-                    LOG.info("successfully connected to remote cache");
+                    log.info("successfully connected to remote cache");
```

Comment:
I don't understand why the logger is renamed from `LOG` to `log` here, where in other files of the PR a new logger named `LOG`. Is being created.

Reply:
It is not renamed, *log* is simply the inherited logger from the super class. That logger is not static final, thus its name is lower case ...

Reply:
Still, it is weird that some classes use uppercase loggers, were this one now uses lowercase.

Reply:
Well, I guess we're just following field naming rules here. Constants are all upper case, instance fields are lower case, right?